docs/content/en/docs/clustermgmt/observability/expose-metrics.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Expose metrics for EKS-A components"
+linkTitle: "Expose metrics"
+weight: 100
+date: 2024-04-06
+description: >
+  Expose metrics for EKS-A components
+---
+
+Some Kubernetes system components like kube-controller-manager, kube-scheduler and kube-proxy expose metrics only on the localhost by default. In order to expose metrics for these components so that other monitoring systems like Prometheus can scrape them, you can deploy a proxy as a Daemonset on the host network of the nodes. The proxy pods also need to be configured with control plane tolerations so that they can be scheduled on the control plane nodes.
+
+To configure a proxy for exposing metrics on an EKS-A cluster, you can perform the following steps:
+
+1.  Create a new cluster role for the client to access the metrics endpoint of the components.
```

Comment:
Why do we need this? I understand the user needs it, but why is it needed to deploy the proxy?

Reply:
It's not required to deploy the proxy. I will move it to another section